LDM-562.tex

```diff
@@ -0,0 +1,357 @@
+
+\textbf{Discussion:}
+Atmospheric distortions should be represented by smoothly-varying function (Gaussian Processes, splines, and polynomials are all worth investigating) that cover the full focal plane with different parameters for each exposure.  It should not be wavelength-dependent within a band (at least, not at a level we will be able to constrain).  Particular care should be taken to avoid overfitting this component, which can turn what otherwise would be a stochastic scatter in positions (in the many-exposure limit) into a systematic error.
+
+\subsection{Astrometric Exposure Pointing Component}
+
+\label{DMS-L2JC-REQ-0016}
+\textbf{ID:} DMS-L2JC-REQ-0016
+
+\textbf{Specification:}
+The JointCal astrometric model shall include an affine transform for each exposure.
+
+\textbf{Discussion:}
+Together with the gnomonic projection, this just is what is typically captured in a typical FITS TAN WCS (albeit applied consistently to all sensors in the focal plane).  All but the offset may be considered part of the atmospheric component instead if the overall scaling and rotation of those components are not normalized.  The offset must remain separate to set the reference point for the gnomonic projection.
+
+\subsection{External Reference Catalogs}
+
+\label{DMS-L2JC-REQ-0017}
+\textbf{ID:} DMS-L2JC-REQ-0017
+
+\textbf{Specification:}
+In both photometric and astrometric fitting, JointCal shall support fitting with external catalogs, which may be different for astrometric and photometric fitting.
+
+\textbf{Discussion:}
+It must be possible to use a trivial reference catalog with no more than two stars (which would constrain only the overall photometric zeropoint and the position, scale, and rotation of the image, requiring internal data to constrain all other terms).  Jointcal must be able to filter reference catalogs on magnitude and color and reject outliers (e.g. due to variability, poor external measurements, or different deblending).
+
+\subsection{Inputs from Colimated Beam Projector}
+
+\label{DMS-L2JC-REQ-0018}
+\textbf{ID:} DMS-L2JC-REQ-0018
+
+\textbf{Specification:}
+JointCal shall support using Collimated Beam Projector data to constrain the photometric fit.
+
+\textbf{Discussion:}
+CBP data consists of measurements of photometric throughput (with no atmospheric contributions) at a particular point on the focal plane at a particular wavelength. Like measurements of stars and galaxies, the overall flux of a particular CBP point is unknown but it can be considered to be constant over all exposures. Photometric fits that utilize CBP data only (i.e. no measurements on science images or reference catalogs) should also be possible for one-off diagnostic runs (i.e. this does not need to be part of normal production interface).
+
+\subsection{Maximum Problem Size}
+
+\label{DMS-L2JC-REQ-0019}
+\textbf{ID:} DMS-L2JC-REQ-0019
+
+\textbf{Specification:}
+JointCal shall be capable of fitting at least 6000 full LSST visits together.
```

Comment:
@RobertLuptonTheGood asks:
> Where in the galaxy?  Do we need a specification on the number of sources per visit?

Since I don't have a good ab-initio estimate on what that number will need to be, I'd prefer to leave it as a design/implementation choice and let it be determined by what we need to meet higher-level requirements on astrometric and photometric accuracy.